Dockerfile

```diff
@@ -29,28 +30,35 @@ FROM rust:alpine${ALPINE_VERSION} as deps
 RUN apk add --no-cache \
         alpine-sdk \
         bash \
+        binutils \
+        ca-certificates \        
         clang-static clang-dev \
-        llvm-static llvm-dev  \
+        cmake \
         git \
+        libc-dev \        
         linux-headers \
+        llvm-static llvm-dev  \
+        nodejs \
```

Comment:
we need to install 16.x

Reply:
On Alpine 3.16, this will always install 16.x